pkgs/development/python-modules/arviz/default.nix

```diff
@@ -31,7 +31,7 @@
 
 buildPythonPackage rec {
   pname = "arviz";
-  version = "0.17.0";
+  version = "0.17.1";
   pyproject = true;
 
   disabled = pythonOlder "3.9";
```

Comment:
```suggestion
  disabled = pythonOlder "3.10";
```

Reply:
Hmm, it seems that the python minimal version is 3.9 for this release:
https://github.com/arviz-devs/arviz/blob/60bbdff87018b907d492d9e228b0d9e67b8a5a2a/setup.py#L61

Reply:
i think maybe python3.9 is dropped from nixpkgs or will be dropped maybe?

Reply:
@GaetanLepage You're right, the minimum was only raised in upstream's main branch.